demosplan/DemosPlanCoreBundle/Services/HTMLSanitizer.php

```diff
@@ -89,6 +89,7 @@ public function purify(string $text): string
         $config = HTMLPurifier_Config::createDefault();
         $config->set('HTML.DefinitionID', 'dplan purifier');
         $config->set('HTML.DefinitionRev', 1);
+        $config->set('Attr.AllowedFrameTargets', ['_blank']);
```

Comment:
To allow `rel` I think you have to add `Attr.AllowedRel` as well

Reply:
done in [058429b](https://github.com/demos-europe/demosplan-core/pull/2531/commits/058429b565a501c6ec554d63eacb56816f040e98)